Library/Homebrew/test/language/node_spec.rb

```diff
@@ -0,0 +1,56 @@
+require "language/node"
+
+describe Language::Node do
+  specify "#npm_cache_config" do
```

Comment:
So, should I add the test for `#npm_cache_config` within the test for `#setup_npm_environment` by checking that `"cache=#{HOMEBREW_CACHE}/npm_cache\n"` is written to the .npmrc file?